mathquill4quill.js

```diff
@@ -106,18 +107,17 @@ window.mathquill4quill = function(dependencies) {
       }
 
       function syncMathquillToQuill(latexInput, saveButton) {
-        const mqField = MathQuill.getInterface(2).MathField(mqInput, {
-          handlers: {
-            edit() {
+        mathQuillConfig.handlers = {
```

Comment:
This will discard any user-provided handlers. I wonder if there is any value in merging-in any existing handlers?

```js
const handlers = mathQuillConfig.handlers == null ? {} : mathQuillConfig.handlers;
mathQuillConfig.handlers = {...handlers, {
  edit() { /* ... */ },
  enter() { /* ... */ }
}};
```

Reply:
I don't know if there is any benefit to it, but it would be good to leave the option open. I just added a commit that should allow for this.